README.rst

```diff
@@ -515,6 +515,43 @@ The script now accepts basic options:
     -D, --debug           Show all messages, including debug messages.
 
 
+Moving records from one collection to another
+---------------------------------------------
+
+Moving records is not directly supported by the CLI tools int this library.
+To do this you'll need to install kinto-wizard.
+
+::
+
+   pip install kinto-wizard
+
+Once kinto-wizard is installed you can then run a dump command in order to
+get all the records from one collection in a file.
+
+::
+
+   kinto-wizard dump \
+      --full --server http://{kinto_url}:8888/v1 \
+      --bucket {bucket_name} \
+      --auth {user}:{password} \
+      > {path_to_file.yaml}
+
+With this local file you can now re-upload all reacords into a new collection
```

Comment:
```suggestion
With this local file you can now re-upload all records into a new collection
```

Reply:
We need to tell the user to edit the file and change the buckets name, groups name, collections names or records ids they want to rename first.

Reply:
Perfect! I'll add those steps!